src/index.md

```diff
@@ -1 +1,81 @@
-# What is Time?
+# Introduction
+
+`time` is a date and time library for Rust. It's:
```

Comment:
Given that nothing comes immediately after "it's", it feels better to say "it is".